internal/distro/test_distro/distro.go

```diff
@@ -14,6 +14,7 @@ import (
 type TestDistro struct {
 	name             string
 	modulePlatformID string
+	ostreeRef        string
```

Comment:
You should also define "ostreeRef" constants for "TestDistro" and "TestDistro2" and set them in `New()` and `New2()` when calling `newTestDistro()`. And as a result please extend the `newTestDistro()` to accept an "ostreeRef" value.